stc_game/Assets/Scripts/Models/DatabaseModels/CharacterBaseModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+[Serializable]
+public class CharacterBaseModel
+{
+    public string Name;
+    public HeroClass HeroClass;
+    public Stats stats;
+    public List<string> Abilities;
```

Comment:
Was just thinking, for this project if we are only going to have a static set of abilities for each class, then we could get away with only storing the HeroClass and not worry about storing a list of ability strings. The game will just create instances of the abilities associated with each class.